core/src/main/java/com/radixdlt/consensus/LedgerHashes.java

```diff
@@ -97,6 +97,8 @@ public final class LedgerHashes {
   private final HashCode transactionRoot;
 
   @JsonProperty("receipt_root")
+  // TODO(resolve during code review): this passes our tests, but shouldn't we exclude it from
+  // consensus currently as well? (see the `stateRoot` above)
```

Comment:
⚠️ This is a very serious question @dhedey @talekhinezh - do we expect non-det only in "resulting substate store", or in the receipt (or rather, its [hashable part](https://github.com/radixdlt/babylon-node/blob/d3096573713051e2fd6c6b60f046d08cfda1c751/core-rust/state-manager/src/types.rs#L221)) too? (I'd prefer to be cautious)

Reply:
The receipt is just as likely to be non-det as the substate changes, so for now, let's not include it in consensus.

We can add them both to consensus at a similar time